.github/workflows/enos-test-ui.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: Vault UI Tests
+
+on:
+  workflow_call:
+    inputs:
+      test_filter:
+        type: string
+        description: "A filter to limit the ui tests to. Will be appended to the ember test command as '-f=<filter>'"
+        required: false
+      storage_backend:
+        type: string
+        description: "The storage backend to use, either 'raft' or 'consul'"
+        default: raft
+
+jobs:
+  run-ui-tests:
+    name: Run UI Tests
+    runs-on: custom-linux-xl-vault-latest
+    timeout-minutes: 60
+    env:
+      GITHUB_TOKEN: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
+      # Pass in enos variables
+      ENOS_VAR_aws_region: us-east-1
+      ENOS_VAR_aws_ssh_keypair_name: ${{ github.event.repository.name }}-ci-ssh-key
+      ENOS_VAR_aws_ssh_private_key_path: ./support/private_key.pem
+      ENOS_VAR_tfc_api_token: ${{ secrets.TF_API_TOKEN }}
+      ENOS_VAR_terraform_plugin_cache_dir: ./support/terraform-plugin-cache
+      ENOS_VAR_vault_license_path: ./support/vault.hclic
+      NODE_VERSION: 16
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get metadata
+        id: get-metadata
+        env:
+          IS_ENT: ${{ startsWith(github.event.repository.name, 'vault-enterprise' ) }}
+        run: |
+          echo "go-version=$(cat ./.go-version)" >> $GITHUB_OUTPUT
+          if ${IS_ENT} == true; then
+            echo "vault_edition=ent" >> $GITHUB_OUTPUT
+            echo "detected vault_edition=ent"
+          else
+            echo "vault_edition=oss" >> $GITHUB_OUTPUT
+            echo "detected vault_edition=oss"
+          fi
+      - name: Set Up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ steps.get-metadata.outputs.go-version }}
+      - name: Configure AWS credentials from Test account
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_CI }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_CI }}
+          aws-region: us-east-1
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN_CI }}
+          role-skip-session-tagging: true
+          role-duration-seconds: 3600
+      - uses: hashicorp/action-setup-enos@v1
+        with:
+          github-token: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
+      - name: Set Up Node
+        uses: actions/setup-node@v2
```

Comment:
I believe `@v3` gets rid of deprecation warnings.

Reply:
done.